### PR TITLE
project_documentation

### PR DESCRIPTION
**Pull request by lascuoana**
### Pull Request Overview

This pull request changes the project documentation.

### Testing Strategy

This pull request was tested by lascuoana.

### TODO or Help Wanted

This pull request still needs a KiCad schematic and the software.

### Build

- [x] Ran `npm build`.

### Author

Signed-off-by: Oana-Roxana Lascu <oana_roxana.lascu@stud.fils.upb.ro>

### CODE REVIEW

File: docs/project/2024/oana_roxana.lascu/index.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+
+| [Car](https://cdn.ozdisan.com/ETicaret_Dosya/729231_182181.pdf) | The chasis | [75 RON](https://www.sigmanortec.ro/Kit-sasiu-Smart-Car-4WD-p136281803) |
+
```
Comment (alexandruradovici) [inline docs/project/2024/oana_roxana.lascu/index.md:78]:
```suggestion
```

File: docs/project/2024/oana_roxana.lascu/index.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+
```
Comment (alexandruradovici) [inline docs/project/2024/oana_roxana.lascu/index.md:76]:
```suggestion
```

File: docs/project/2024/oana_roxana.lascu/index.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+| [Car](https://cdn.ozdisan.com/ETicaret_Dosya/729231_182181.pdf) | The chasis | [75 RON](https://www.sigmanortec.ro/Kit-sasiu-Smart-Car-4WD-p136281803) |
+
+| [L298N Dual H-Bridge Motor Driver](https://components101.com/sites/default/files/component_datasheet/L298N-Motor-Driver-Datasheet.pdf) | The motor driver | [9 RON] (https://www.optimusdigital.ro/ro/drivere-de-motoare-cu-perii/145-driver-de-motoare-dual-l298n.html?search_query=l298n&results=4)
+
```
Comment (alexandruradovici) [inline docs/project/2024/oana_roxana.lascu/index.md:80]:
```suggestion
```

File: docs/project/2024/oana_roxana.lascu/index.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+| [IR sensor modules](https://www.circuits-diy.com/hw201-infrared-ir-sensor-module/) | The infrared sensors | [7 RON]
+(https://www.optimusdigital.ro/ro/senzori-senzori-optici/4514-senzor-infrarosu-de-obstacole.html?search_query=senzor+infrarosu&results=121)
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/oana_roxana.lascu/index.md:90]: Do you need these libraries?

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill the pull request template.

File: docs/project/2024/oana_roxana.lascu/index.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
+# Line Following Car
+
```
Comment (alexandruradovici) [inline docs/project/2024/oana_roxana.lascu/index.md:2]: Please add a one line description of your project.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by alexandruradovici** [comment]
Please fill the pull request template.

File: docs/project/2024/oana_roxana.lascu/index.md (hunk 4 above)
Comment (lascuoana) [reply docs/project/2024/oana_roxana.lascu/index.md]: I didn't realize that the software part wasn't edited, but I fixed it.

File: docs/project/2024/oana_roxana.lascu/index.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
+# Line Following Car
+Using a 4wd smart car chasis, I will attach two IR sensors and a motor driver to the Raspberry Pi Pico W board, and I will use a pid algorithm in Rust, in order to build a line following car.
```
Comment (alexandruradovici) [inline docs/project/2024/oana_roxana.lascu/index.md:2]: Please describe your project briefly here. *A car that does this and that*.

File: docs/project/2024/oana_roxana.lascu/index.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+| [Car](https://cdn.ozdisan.com/ETicaret_Dosya/729231_182181.pdf) | The chasis | [75 RON](https://www.sigmanortec.ro/Kit-sasiu-Smart-Car-4WD-p136281803) |
+| [L298N Dual H-Bridge Motor Driver](https://components101.com/sites/default/files/component_datasheet/L298N-Motor-Driver-Datasheet.pdf) | The motor driver | [9 RON] (https://www.optimusdigital.ro/ro/drivere-de-motoare-cu-perii/145-driver-de-motoare-dual-l298n.html?search_query=l298n&results=4)
+| [IR sensor modules](https://www.circuits-diy.com/hw201-infrared-ir-sensor-module/) | The infrared sensors | [7 RON]
+(https://www.optimusdigital.ro/ro/senzori-senzori-optici/4514-senzor-infrarosu-de-obstacole.html?search_query=senzor+infrarosu&results=121)
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [PID Controller](https://github.com/topics/pid-controller?l=rust&o=desc&s=updated) | Implements a PID (Proportional-Integral-Derivative) controller. | Used for error correction|
```
Comment (alexandruradovici) [inline docs/project/2024/oana_roxana.lascu/index.md:85]: This is not a library/

File: docs/project/2024/oana_roxana.lascu/index.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+| [rand](https://github.com/engusmaze/frand) | Generates random numbers | Introduces randomness in motor adjustments (e.g., slight variations in speed or turning angles) to simulate real-world conditions and prevents the car from following the same path exactly every time. |
+| [time](https://github.com/time-rs/time) |Provides time-related functionality. |Measures time intervals for calculating integral and derivative terms in the PID controller and ensures stable control loops with accurate timing.|
+| [rp2040_hal](https://github.com/rp-rs/rp-hal) |Board support for Raspberry Pi Pico  |Configures pins and peripherals on the Pico (e.g., connecting IR sensors, motor driver) and enables communication with other components. |
+| [elf2uf2-rs](https://github.com/rej696/elf2uf2) |Rust port of ELF2UF2 for flashing programs |Loads compiled firmware onto the Raspberry Pi Pico W, essential for deploying the code to the board|
```
Comment (alexandruradovici) [inline docs/project/2024/oana_roxana.lascu/index.md:89]: This is not a library.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by ZamfirAndreiPaul — APPROVED** [review]

**Pull request merged**